src/index.js

```diff
@@ -3,10 +3,17 @@ import ReactDOM from "react-dom";
 
 import "./styles.css";
 import { Game } from "./Game";
+import { BOARD_SIZE_MAX, BOARD_SIZE_MIN } from './constants';
 
-const App = function() {
-  const boardSizeX = 10;
-  const boardSizeY = 10;
+const App = function () {
+  // eslint-disable-next-line
+  let size = parseInt(new URLSearchParams(location.search).get('size'), 10);
```

Comment:
Instead of one big line, break into separate separate smaller ones.

```js
const strQuery = window.location.search;
const paramsQuery = new URLSearchParams(strQuery);
const paramSize = paramsQuery.get('size');

let size = parseInt(paramSize, 10);
```